src/main/java/nl/tudelft/semgroup4/collision/DefaultCollisionHandler.java

```diff
@@ -230,4 +248,34 @@ public void run() {
         Game.LOGGER.log(LogSeverity.DEBUG, "Collision", "Player picked up a utility");
         util.activate(game.getCurLevel());
     };
+    
+    private final CollisionHandler<HorMovingWall, AbstractWall> horWallWallHandler = 
+            (game, movingWall, abstractWall) -> {
+        int offset = 5;        
+        
+        if (movingWall.getLocX() < abstractWall.getLocX() 
```

Comment:
My goal with this is to define a general formula to determine the off-set for any object. If the off-set is always 2*speed of the object, then we could make a small util like function that can be called for every object to determine it. Then you could define methods like isBelow(objA, objB) and make that return the desired value for the expression that you now do manually.

Of course, if every object / scenario where this applies has a different expression or off-set, this is not useful.

I think you are the best judge of this, as you implemented all these off-sets.

EDIT: I see you edited this diff, so we can leave this at that.

< logic, organisation